app/views/users/emails/verify.html.erb

```diff
@@ -1,17 +1,9 @@
 <% self.title = t('titles.verify_email') %>
 
-<% if @resend_confirmation %>
-  <%= render AlertComponent.new(
-        type: :success,
-        class: 'margin-bottom-4',
-        message: t('notices.resend_confirmation_email.success'),
-      ) %>
-<% end %>
-
```

Comment:
The `@resend_confirmation` instance variable doesn't exist in the context of adding an email to an existing account. The success banner is shown as a result of the `Users::EmailsController#resend` action [`flash[:success]` assignment](https://github.com/18F/identity-idp/blob/c3112740fb2ede6f2e9fdd796128b84e1874b8ad/app/controllers/users/emails_controller.rb#L44), with test coverage added in this branch.